docs/steering-committee/index.md

```diff
@@ -1,6 +1,6 @@
-# Steering Commitee Meetings
+# Steering Committee Meetings
 
-Here you will find the meeting minutes from the Steering Committiee and
-Community Meetings.
+Here you will find the meeting minutes from the Steering Committiee
```

Comment:
```suggestion
Here you will find the meeting minutes from the Steering Committee
```

Are there any spellcheckers that can be added to the CI?

Reply:
Currently there is one, `codespell`, from what I can see it only works by looking for explicitly defined misspelled words (common misspellings). It suggested `Commitee` -> `Committee` change in the same file.
If you want to add/change to different spellchecker then maybe create new issue here or in https://github.com/3mdeb/hooks so we could decide on best one that can be used everywhere.